generators/app/index.js

```diff
@@ -9,27 +9,35 @@ var semver = require('semver');
 var versions = require('./dependency-versions.js');
 var yosay = require('yosay');
 var _ = require('lodash');
+var genConfig = null;
```

Comment:
While Javascript has ```null``` it also has ```undefined```. Generally we use ```undefined``` to mean that a value has not been provided yet. ```null``` is typically used to indicate an explicitly unset state. In this case, I think you mean ```undefined```.